gui/default/syncthing/core/syncthingController.js

```diff
@@ -549,10 +567,28 @@ angular.module('syncthing.core')
                 $scope.connectionsTotal = data.total;
 
                 data = data.connections;
+                var currentAddresses = [];
```

Comment:
Couldn't this be a map? Then the check below could be a simple lookup instead of an `includes` call (which I assume is a shortcut for a loop). Not that it matters much, but I'd consider that as simpler.

Reply:
Got rid of this altogether, by just rebuilding the map from the previously cached or newly probed results.